airflow/providers/amazon/aws/operators/datasync.py

```diff
@@ -307,18 +317,22 @@ def _create_datasync_task(self):
         if not self.task_arn:
             raise AirflowException("Task could not be created")
         self.log.info("Created a Task with TaskArn %s", self.task_arn)
-        return self.task_arn
```

Comment:
This returned value is not used anywhere and `_create_datasync_task` is private method.